caselaw.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>Caselaw Access Project</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="robots" content="noindex" />
+		<meta name="author" content="Your name" />
+		<meta name="description" content="Explore American Caselaw" />
+		<meta property="og:title" content="Caselaw Access Project" />
+		<meta property="og:description" content="Explore American Caselaw" />
+		<meta property="og:url" content="https://case.law/" />
+		<meta property="og:type" content="article" />
+		<meta property="og:site_name" content="Caselaw Access Projects" />
+
+		<link href="/css/global.css" rel="stylesheet" />
+		<script type="module" src="/components/cap-map.js"></script>
+		<script type="module" src="/components/cap-header.js"></script>
+		<script type="module" src="/components/cap-stats.js"></script>
+		<script type="module" src="/components/cap-section-highlight.js"></script>
+		<script type="module" src="/components/cap-contact.js"></script>
+		<script type="module" src="/components/cap-footer.js"></script>
+		<script type="module" src="/components/cap-content-router.js"></script>
+		<script type="module" src="/components/cap-case.js"></script>
+		<script type="module" src="/components/cap-volume.js"></script>
+		<script type="module" src="/components/cap-jurisdictions.js"></script>
+		<script type="module" src="/components/cap-reporter.js"></script>
+
+		<script>
+			var BUCKET_ROOT = "https://cap-redacted-demo.s3.us-west-2.amazonaws.com";	
```

Comment:
> @mdellabitta: The entry point for this code is in caselaw.html, which primarily is a copy of the original index.html and I have very little attachment to. It primarily loads each of the web components that might be used to render the content, performs a little layout (totally TBD), and sets a global variable BUCKET_ROOT that sets the base path that fetches will come from.
> Someone with better knowledge of vanilla JS projects might know of a better way to inject properties like this.

I don't know if it's any better, but you could alternatively consider using `window.CAP_BUCKET_ROOT` to simplify questions related to scoping. 

---

More "proper" but adding a roundtrip, you might consider creating a data module for this under `/data` and importing it at component-level, as @tinykite does.